### PR TITLE
Add libatasmart package

### PR DESCRIPTION
**Pull request by Zopolis4**
## Description
Adds the libatasmart package (part of the same larger package as #6766).

## Additional information
Works properly:
- [x] armv7l [libatasmart-0.19-chromeos-armv7l.tar.zip](https://github.com/skycocker/chromebrew/files/8189862/libatasmart-0.19-chromeos-armv7l.tar.zip)

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/Zopolis4/chromebrew.git CREW_TESTING_BRANCH=libatasmart CREW_TESTING=1 crew update
```
---

### CODE REVIEW

**Comment by satmandu** [comment]
No dependencies?

**Comment by Zopolis4** [comment]
> No dependencies?

None that I could see, although I do have a lot of programs installed at this point and dont really have the setup to test for dependencies.

File: packages/libatasmart.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class Libatasmart < Package
+  description 'LIBATASMART ATA S.M.A.R.T. Reading and Parsing Library'
+  homepage 'http://git.0pointer.net/libatasmart.git/'
+  version '0.19'    
+  license 'LGPL-2.1+'
+  compatibility 'all'
+  source_url 'http://git.0pointer.net/libatasmart.git/'
+  git_hashtag "v#{version}"
+  git_branch 'master'
```
Comment (uberhacker) [inline packages/libatasmart.rb:11]: I don't think you need the git branch if a git tag is provided?

Comment (satmandu) [reply packages/libatasmart.rb:11]: Needed for broken older git servers as a workaround for `fatal: dumb http transport does not support shallow capabilities` issue.

**Pull request merged**